src/Components/Publishing/Layouts/ArticleLayout.tsx

```diff
@@ -52,18 +52,22 @@ interface ArticleContainerProps {
   marginTop?: string
 }
 
-@track((props) => {
-  return {
-    page: "Article",
-    entity_type: "article",
-    entity_id: props.article.id
-  }}, {
-    dispatch: data => Events.postEvent(data)
+@track(
+  props => {
+    return {
+      page: "Article",
+      entity_type: "article",
+      entity_id: props.article.id,
+    }
+  },
+  {
+    dispatch: data => Events.postEvent(data),
   }
 )
 export class ArticleLayout extends React.Component<ArticleProps, ArticleState> {
   static childContextTypes = {
     onViewFullscreen: PropTypes.func,
+    tracking: PropTypes.object,
```

Comment:
Since tracking injects a context, and we were requesting a context for `onFullScreen`, without defining it in the list it errors out.